src/controllers/CardsController.ts

```diff
@@ -42,5 +42,17 @@ export class CardsController {
 					.then(response => response && res.status(200).send(response));
 			}
 		);
+
+		app.put(
+			'/:id',
+			basicAuth({ users: { admin: 'admin' } }),
+			(req: Request, res: Response) => {
+				const { id } = req.params;
```

Comment:
We can reduce this down to one line to clean it up

`const { body, params: { id }  } = params;`